README.md

```diff
@@ -312,6 +312,35 @@ Now point your browser to:  `http://<IP:NodePort>`, enable your **developer mode
 
 > Note: using fault injection or mixer rule won't able to trigger the circuit breaker because all the traffic will be aborted/delayed before it get sent to the cloudant's Envoy.
 
+Next, let the circuit breaker eject any broken pod in you load balancing pool to avoid further failures. First, create a new cloudant database, cloudant-db2, that listens to the wrong host.
```

Comment:
what is a pool in the kubenretes conext ?  then we are going to add the two database instances in one pool. each instance is a represented by a pod of two containers (database container, sidecar container)